.travis.yml

```diff
@@ -25,13 +25,6 @@ matrix:
     sudo: required
     dist: trusty
     services: docker
-    env: BEAKER_set="debian-6"
```

Comment:
I'm testing leaving out the ldif-wrap=no thing for now by commenting it out. The rest of the code should be unaffected anyways. You can add it back in as soon as your servers are migrated.

If you have an idea on how to check (in the custom provider) whether it's running on Squeeze, I'd be happy to add it as well. I didn't have the time to test that out, though.